.ci/Jenkinsfile

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env groovy
```

Comment:
new pipeline for the new job, it launches downstream jobs per Python version, instead of thousand of stages